templates/docs.hamlet

```diff
@@ -0,0 +1,121 @@
+<h1>Carnival Documentation
+
+<article>
+  <pre>
+    <code>
+      \&lt;script src='#{root}/YOUR-SITE-ID/init.js'&gt;&lt;/script&gt;
+      \&lt;script&gt;
+      \  document.addEventListener("DOMContentLoaded", Carnival.init);
+      \&lt;/script&gt;
+
+  <h2>Options
+
+  <p>
+    <code>Carnival.init
+    accepts an options object with any of the following keys:
+
+  <table .table>
+    <tr>
+      <th>Option
+      <th>Default Value
+      <th>Description
+    <tr>
+      <td><code>enabled</code>
+      <td><code>true</code>
+      <td>Add Carnival on this page?
+    <tr>
+      <td><code>article_author</code>
+      <td><code>""</code>
+      <td>The Article author. Used in the Comments Feed.
+    <tr>
+      <td><code>article_selector</code>
+      <td><code>"article"</code>
+      <td>The CSS selector used to find the Article element.
+    <tr>
+      <td><code>block_selector</code>
+      <td><code>":scope > p, :scope > pre"</code>
```

Comment:
Oh, neat: https://developer.mozilla.org/en-US/docs/Web/CSS/:scope

How about linking to those docs?